src/pages/K8s.vue

```diff
@@ -156,6 +169,9 @@ export default {
     cannotReset() {
       return ![K8s.State.STARTED, K8s.State.ERROR].includes(this.state);
     },
+    cannotModifyUseOfKubernetes() {
```

Comment:
This is identical to `cannotReset` — I guess it's just here because it's semantically different?

Reply:
They started off different, but ended up the same, and I didn't notice.